index.html

```diff
@@ -26,9 +26,10 @@
     <div class="reveal">
       <div class="slides">
         <section id="qrcode" data-menu-title="QR code">
-          <div class="qrcode" id="qrcode-talk"/>
```

Comment:
Yeah but the correct rendering **is** actually (2). QR code at the top, then the link, then whatever comes after the link. If you want it the other way around (link first), then the correct HTML would be
```html
<section id="qrcode" data-menu-title="QR code">
  <p><a href="https://example.com/link/to/this/talk"
        target="_blank"
        id="talk">https://example.com/link/to/this/talk</a></p>
  <div class="qrcode" id="qrcode-talk"></div>
</section>
```
So I do totally concede that the code as it is is broken and doesn't actually do what I originally intended it to do (but then forgot about), which is to render the link **below** the QR code, as a caption. But the correct fix is to properly close the `<div>` where it is, not to wrap it around the link.